tests/src/oneof_attributes.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package Foo.Custom.OneOfAttrs;
```

Comment:
Is it necessary for this to be in a separate file/package than custom_attributes.proto?  If not, I think it would be good to consolidate.  If there's some reason to have them separate than maybe a comment explaining would be good.

Reply:
I added the explanation. Unfortunately, the companion module for an oneof field triggers the missing-docs error and I see no way how to place docs on that one (I don't think it makes sense to extend the support of the custom attributes to that kind of module).

Reply:
Makes sense, thanks for clarifying.